sdks/python/apache_beam/io/filesystem.py

```diff
@@ -96,14 +102,19 @@ def mime_type(cls, compression_type, default='application/octet-stream'):
         cls.BZIP2: 'application/x-bz2',
         cls.DEFLATE: 'application/x-deflate',
         cls.GZIP: 'application/x-gzip',
+        cls.ZSTD: 'application/x-zstd',
```

Comment:
nit: I think it's `application/zstd`
https://datatracker.ietf.org/doc/html/rfc8478 
https://datatracker.ietf.org/doc/html/rfc6648#appendix-B